test/WebJobs.Script.Tests/TestScripts/Node/HttpTriggerExpressApi/index.js

```diff
@@ -0,0 +1,20 @@
+﻿var util = require('util');
+
+module.exports = function (context, req) {
+    context.log('Node.js HttpTrigger function invoked.');
+
+    context.res.status(200)
+        .set('test-header', 'Test Response Header')
+        .set('Content-Type', 'application/json; charset=utf-8')
+        .send({
```

Comment:
In Express, send will actually send the response, right? Whereas here, this is just a builder, and we don't actually send the response until much later, after context.done() is called, and all output bindings are processed?

Should we tie send/end to context.done?